docs/configuration.asciidoc

```diff
@@ -1269,7 +1269,7 @@ Example: `0ms`.
 [[config-central-config]]
 ==== `central_config` (added[1.8.0])
 
-When enabled, the agent will make periodic requests to the APM Server to fetch updated configuration.
+When enabled, the agent will make periodic requests to the APM Server (every 5 minutes) to fetch updated configuration.
```

Comment:
It's not 5 minutes. The logic is to check the "Cache-Control" header from the central config, and use the "max-age" value if set. If not set, then the 5 minute default is used. The cache control header max age from the central config looks like it is set to 30 seconds

Reply:
🤦 Right - it was on the same file... https://github.com/elastic/apm-agent-java/blob/9a12b98ee959abb4a596c823325d9c78d827724b/apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/ApmServerConfigurationSource.java#L96

Reply:
I've proposed an update to the statement.